packages/generator/tests/utilities/measurements/non_streaming_data_measurement/_stubs/color.proto

```diff
@@ -0,0 +1,21 @@
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+package ni.measurementlink.measurement.sample_measurement.v1;
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+option csharp_namespace = "NationalInstruments.MeasurementServices.Measurements";
```

Comment:
Given that this proto will be used only for Python tests, should we remove this?

Reply:
Sure removing it.